src/interactions/slashCommands/SlashCommandBuilder.ts

```diff
@@ -22,6 +27,12 @@ export class SlashCommandBuilder {
 	 */
 	public readonly options: ToAPIApplicationCommandOptions[] = [];
 
+	/**
+	 * Whether the command is enabled by default when the app is added to a guild
+	 * @default true
+	 */
+	public readonly defaultPermission: boolean = true;
```

Comment:
Honestly, we should keep it as undefined unless its mandatory

Reply:
But the default *is* `true` though? https://discord.com/developers/docs/interactions/application-commands#application-command-object-application-command-structure

Reply:
But it's not mandatory.